modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -390,9 +452,15 @@ const ActionForm = ({
 										})
 									}
 									options={
-										Liferay.FeatureFlags['LPS-194395']
-											? ACTION_TYPES
-											: ACTION_TYPES.slice(0, 1)
+										activeTab === 0
+											? Liferay.FeatureFlags['LPS-194395']
+												? ITEM_ACTION_TYPES.concat(
+														ACTION_TYPES
+												  )
```

Comment:
We can avoid code duplication here, by extracting concatenation to definition. Something like:
```js
const ITEM_ACTION_TYPES = [
	...
].concat(ACTION_TYPES);
```